resources/assets/icons/danger.svg

```diff
@@ -0,0 +1 @@
+<svg version="1.1" id="Слой_1" xmlns="http://www.w3.org/2000/svg" x="0" y="0" viewBox="0 0 11.8 11.8" xml:space="preserve"><style>.danger-st0{fill:none;stroke:currentColor;stroke-width:2;stroke-linecap:round;stroke-linejoin:round}</style><g id="danger_2" transform="translate(-453.086 -4389.086)"><g id="remove-circle"><path id="Shape_1779" class="danger-st0" d="M454.5 4399.5l9-9"/><path id="Shape_1780" class="danger-st0" d="M463.5 4399.5l-9-9"/></g></g></svg>
```

Comment:
we already have `cross` and `close` that are similar, should it replace one of those?

Reply:
Just need to delete the `cross` one and we keep the rest according to Oleg.